.github/workflows/west_cmds.yml

```diff
@@ -55,7 +55,7 @@ jobs:
     - name: install pytest
       run: |
         pip3 install wheel
-        pip3 install pytest west pyelftools canopen progress mypy intelhex
+        pip3 install pytest west pyelftools canopen progress mypy intelhex psutil
```

Comment:
psutil doesn't exist on Windows, don't think this can be made a generic dependency.

Reply:
`psutil` exists on Windows, here is proof from official documentation:
https://psutil.readthedocs.io/en/latest/#about

Moreover there is west command test performed bu github workfow with windows:
https://github.com/evgeniy-paltsev/zephyr/runs/1249826436
It was run from my branch with this change.

Reply:
missed that, thanks for showing the proof!